x/mint/README.md

```diff
@@ -47,6 +46,56 @@ oneYear := time.Hour * 24 * 365
 | 19   | 1.50              |
 | 20   | 1.50              |
 
+## State
+
+See [./types/minter.go](./types/minter.go) for the `Minter` struct which contains all of the state of this module.
+
+## State Transitions
+
+The `Minter` struct is updated every block via `BeginBlocker`.
+
+## Begin Block
```

Comment:
```suggestion
### Begin Block
```
I feel like this should be a sub header of state transitions (maybe also events as well)